### PR TITLE
Python fixes

### PR DESCRIPTION
**Pull request by codeFather2**

### CODE REVIEW

File: Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs [hunk 1]
```diff
@@ -44,6 +44,14 @@ public Ust VisitFile_input(PythonParser.File_inputContext context)
             {
                 block.Statements.Add(Visit(child).ToStatementIfRequired());
             }
+
+            if (root.CurrentSourceFile?.Name.Length > 0)
+            {
+                var dotIndex = root.CurrentSourceFile.Name.LastIndexOf('.');
+                return new NamespaceDeclaration(new StringLiteral(root.CurrentSourceFile?.Name
```
Comment (gabriel-fallen) [inline Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs]: Could you please reformat this like
```cs
return new NamespaceDeclaration(
    new StringLiteral(root.CurrentSourceFile?.Name.Remove(dotIndex < 0 ? 0 : dotIndex)),
    block,
    context.GetTextSpan());
```
? Otherwise it's hard to follow where one argument ends and the next one starts.

File: Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs [hunk 2]
```diff
@@ -984,12 +992,20 @@ public Ust VisitEncoding_decl(PythonParser.Encoding_declContext context)
 
         public Ust VisitYield_expr(PythonParser.Yield_exprContext context)
         {
-            return VisitChildren(context);
+            var yielArg = context.yield_arg();
+            return yielArg == null
+                ? new YieldExpression(null, context.GetTextSpan())
+                : new YieldExpression(Visit(context.yield_arg()).ToExpressionIfRequired(), context.GetTextSpan());
```
Comment (gabriel-fallen) [inline Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs]: I think you can reuse `yielArg` here. And rename it to `yieldArg` by the way. :)

**Review by gabriel-fallen — APPROVED** [review]
Looks about right. :smile:

**Review by KvanTTT — APPROVED** [review]
